lm_engine/modeling_utils/sequence_mixer_blocks/softmax_attention/flash_attention.py

```
@@ -142,8 +153,11 @@ def flash_attention(
             softcap=softcap,
             forward_function=_flash_attention_forward,
             backward_function=_flash_attention_backward,
+            deterministic=deterministic,
```

Ruff (0.16.1)

[error] Undefined name `deterministic`

(F821)